Tiling/gdal2tiles_parallel.py

```diff
@@ -1196,7 +1213,9 @@ def open_input(self):
                     # Generation of VRT dataset in tile projection, default 'nearest neighbour' warping
                     self.out_ds = gdal.AutoCreateWarpedVRT(
                         self.in_ds, self.in_srs_wkt,
-                        self.out_srs.ExportToWkt())
+                        self.out_srs.ExportToWkt(),
+                        self.resampling,
```

Comment:
I am getting a `TypeError` when AutoCreateWarpedVRT is called with these extra params.  What versions of GDAL and the Python bindings are you using?